### PR TITLE
WRO-540: TypeScript: Update @enact/template-typescript to sandstone

### PR DESCRIPTION
**Pull request by vladut-clapou-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated @enact/template-typescript to use sandstone instead of moonstone

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-540

### Comments
Enact-DCO-1.0-Signed-off-by: Clapou George-Vladut(vladut.clapou@lgepartner.com)

### CODE REVIEW

File: packages/typescript/template/src/App/App.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import kind from '@enact/core/kind';
-import MoonstoneDecorator from '@enact/moonstone/MoonstoneDecorator';
-import Panels from '@enact/moonstone/Panels';
+import ThemeDecorator from '@enact/sandstone/ThemeDecorator';
+import Panels from '@enact/sandstone/Panels';
```
Comment (daniel-stoian-lgp) [inline packages/typescript/template/src/App/App.tsx]: sort

File: packages/typescript/template/package.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
   "dependencies": {
     "@enact/core": "^4.0.0",
     "@enact/i18n": "^4.0.0",
-    "@enact/moonstone": "^4.0.0",
+    "@enact/sandstone": "2.1.4",
```
Comment (daniel-stoian-lgp) [inline packages/typescript/template/package.json]:
```suggestion
    "@enact/sandstone": "^2.1.4",
```

File: packages/typescript/template/package.json [hunk 3]
```diff
@@ -32,7 +32,7 @@
   "dependencies": {
     "@enact/core": "^4.0.0",
```
Comment (daniel-stoian-lgp) [inline packages/typescript/template/package.json]: update all other dependecies

File: packages/typescript/template/src/views/MainPanel.tsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
-import Button from '@enact/moonstone/Button';
+import Button from '@enact/sandstone/Button';
 import kind from '@enact/core/kind';
-import {Panel, Header} from '@enact/moonstone/Panels';
+import {Panel, Header} from '@enact/sandstone/Panels';
```
Comment (daniel-stoian-lgp) [inline packages/typescript/template/src/views/MainPanel.tsx]: sort

File: packages/typescript/template/package.json [hunk 5]
```diff
@@ -30,18 +30,18 @@
     "dist/*"
   ],
   "dependencies": {
-    "@enact/core": "^4.0.0",
-    "@enact/i18n": "^4.0.0",
-    "@enact/moonstone": "^4.0.0",
-    "@enact/spotlight": "^4.0.0",
-    "@enact/ui": "^4.0.0",
+    "@enact/core": "^4.0.11",
+    "@enact/i18n": "^4.0.11",
```
Comment (daniel-stoian-lgp) [inline packages/typescript/template/package.json]: latest version is 4.14

**Review by daniel-stoian-lgp — APPROVED** [review]

File: packages/typescript/template/package.json [hunk 6]
```diff
@@ -30,19 +30,19 @@
...
-    "@enact/core": "^4.0.0",
-    "@enact/i18n": "^4.0.0",
-    "@enact/moonstone": "^4.0.0",
-    "@enact/spotlight": "^4.0.0",
-    "@enact/ui": "^4.0.0",
+    "@enact/core": "^4.1.4",
+    "@enact/i18n": "^4.1.4",
+    "@enact/sandstone": "^2.1.4",
+    "@enact/spotlight": "^4.1.4",
+    "@enact/ui": "^4.1.4",
```
Comment (juwonjeong) [inline packages/typescript/template/package.json]:
```suggestion
    "@enact/core": "^4.5.0-alpha.2",
    "@enact/i18n": "^4.5.0-alpha.2",
    "@enact/sandstone": "^2.5.0-alpha.2",
    "@enact/spotlight": "^4.5.0-alpha.2",
    "@enact/ui": "^4.5.0-alpha.2",
```

**Review by juwonjeong — APPROVED** [review]
LGTM

**Pull request merged**